pkg/store/proxy_test.go

```diff
@@ -1138,6 +1144,7 @@ func TestProxyStore_LabelNames(t *testing.T) {
 				nil,
 				nil,
 				func() []Client { return tc.storeAPIs },
+				nil,
```

Comment:
I think here would be actually the correct place to test it, no?

Reply:
sounds good :+1:

Reply:
Ah i understand this area now better. Well, I explicitely wanted to test storeset handling but I can add an more thorough test here indeed.